src/libs/ReimbursementAccountUtils.js

```diff
@@ -36,10 +37,22 @@ function getErrorText(props, errorTranslationKeys, inputKey) {
     return errors[inputKey] ? props.translate(errorTranslationKeys[inputKey]) : '';
 }
 
+/**
+ * Helper method to get the onfido sdk token from the bank account's achdata
+ *
+ * @param {Object} achData
+ * @returns {string}
+ */
+function getOnfidoSDKTokenFromACHData(achData) {
+    const sdkToken = _.get(achData, ['verifications', 'externalApiResponses', 'requestorIdentityOnfido', 'apiResult', 'sdkToken'], '');
+    return sdkToken || _.get(achData, ['sdkToken'], '');
```

Comment:
```suggestion
    return sdkToken;
```

The sdkToken should always be in under verifications now, right?

Reply:
Yes, but it will break for bank accounts that are already there. 
Maybe we don't care b/c signing out and back in should fix it?